fetch/api/basic/historical.any.js

```diff
@@ -0,0 +1,16 @@
+test(() => {
+  assert_false("getAll" in new Headers);
+  assert_false("getAll" in Headers.prototype);
+}, "Headers object no longer has a getAll() method");
+
+test(() => {
+  const request = new Request("https://domfarolino.com");
```

Comment:
Using random domains makes me nervous, even if they shouldn't be contacted (it probably can't happen here but sometimes e.g. prefetching ends up causing forbidden requests). Can we just use the wpt domain instead?

Reply:
Sure.